src/Components/Popover/Popover.js

```diff
@@ -133,6 +138,17 @@ const Popover = props => {
     close();
   };
 
+  React.useEffect(() => {
+    if (showing && closeAfter) {
+      const timer = setTimeout(() => {
+        close();
+      }, closeAfter);
```

Comment:
```suggestion
      const timer = setTimeout(close, closeAfter);
```

Reply:
Same comment as before. in the event of a formatting preference please create a lint rule to address.